contributing/releasing.md

```diff
@@ -15,8 +15,8 @@ Each release is driven by a single **release engineer**, who is also a Googler.
 is expected to do the following, in order of priority:
 
 - Do not break Google.
-- Cut a release by first thing Wednesday morning, NYC time, without exception.
-- Land a release at least once a week.
+- Cut a release as early in the working calendar week as possible, NYC time, without exception.
```

Comment:
Would Wednesday releases now be considered an exception according to this definition?

Reply:
I softened this line a bit. I think the second line's "Land a release at least once every calendar week." is sufficient to emphasize the importance of landing a release once a week.